src/news-update/index.md

```diff
@@ -0,0 +1,17 @@
+# What is a news update?
+
+News updates are a great way to garner your game some new or renewed publicity. They are the primary means of staying in contact with your game's followers. Post news updates whenever something interesting happens with your game. Followers will be sent notifications and the news will be archived on your game page.
+
+The best news updates deliver information that a game's followers would want to know. If you upload a new version of your game, hit a major milestone on development, have updated changelogs or teasers, you should take the time to make a news update.
+
+## How do I post a news update?
+
+The best news updates are occasional and contain interesting and important information. Embedded images go a long way in keeping readers interested.
+
+The content is written in Markdown, which is easy to use and allows you to do a number of cool things without having to use HTML. For help with Markdown, consult our [Markdown guide](https://help.gamejolt.com/markdown).
```

Comment:
work this into the next paragraph so that we're not referencing Markdown Guide so much

"+We recommend using [Imgur](http://imgur.com/) to host the images for your news update. You can embed screenshots and game art using Markdown. Although video is currently not an option via Markdown, you can always add animated gifs!"